data/systemd/snapd.apparmor.service.in

```diff
@@ -8,7 +8,6 @@
 Description=Load AppArmor profiles managed internally by snapd
 DefaultDependencies=no
 Before=sysinit.target
-Requisite=snapd.service
```

Comment:
tl;dr - this change LGTM. +1

I'm not sure why `Requisite=snapd.service` was ever used-- this unit should not care anything about snapd-- it just needs to load the snapd apparmor profiles (perhaps `RequisiteOf=snapd.service` would be ok, but I don't see the point in failing this unit if snapd failed-- there might be daemon snaps, etc on the system that should not fail to start simply because snapd failed to start (which they would if the apparmor profile wasn't loaded for it, since snap-confine would not be able to change profile into it).

I did recommend considering `Requisite=apparmor.service` here: https://github.com/snapcore/snapd/pull/5198#discussion_r190575058. This is what we want to achive:
* only start this unit if apparmor is available in the kernel (`ConditionSecurity=apparmor`)
* make sure this unit is started after the apparmor unit (`After=apparmor.service`)

The idea behind `Requisite=apparmor.service` from the other PR is that there is no point starting this unit if the apparmor.service failed. This idea was not tested (at least by me), but I now recall that the apparmor.service will show as failed if *any* profile failed to load. I think this would be far too brittle because it means, for example, if the admin made a syntax error in some non-snap apparmor profile, then on reboot the apparmor.service would load all the profiles it could, but find the error in the errant profile and exit with error, and then all the snap policy would not be loaded. Therefore, `After=apparmor.service` and `ConditionSecurity=apparmor` should be good enough.